person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
```

Comment:
- [ ] Kindly make sure the `Person` has many `rentals`
For example, You should have:
```
attr_accessor :name, :age, :rentals
   
  def initialize(age, name = "Unknown", parent_permission = true)
    .
    .    @rentals = []
  end

  *A method for adding rentals should be present. 
  def add_rental(book, date)
    *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```

Reply:
- [ ] Kindly re-check the project [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md) and add the required classes and their associations.
For example:
- Create a class Classroom with the following:
   - @label instance variable, should be initialized in the constructor.
   - Setter and getter for @label (remember about attr_accessor).
- Create the has-many/belongs-to relationship between Classroom and Student. The following should be implemented:
   - Create the has-many side (a classroom has many students).
   - Create the belongs-to side (a student belongs to a classroom).
   - Make sure that when adding a student to a classroom it also sets the classroom for the student.
   - Make sure that when setting the classroom for a student it also adds it to the classrooms' students.
- Create a class Book with the following:
   - @title and @author instance variables, should be initialized in the constructor.
   - Setters and getters for instance variables (remember about attr_accessor).

Reply:
thank you this has been implimented